AdobeStockImageAdminUi/Test/Mftf/Section/AdobeStockSection.xml

```diff
@@ -21,7 +21,8 @@
         <element name="modalError" type="text" selector="//aside[contains(@class, 'modal-popup')]//div[@class='modal-content']//div[text()='Something went wrong.']"/>
         <element name="countGridImages" type="text" selector="//div[@class='masonry-image-grid']/div[contains(.,data-repeat-index)]"/>
         <element name="recordsFound" type="text" selector="//*[@id='adobe-stock-images-search-modal']//*[text()='records found']/parent::div"/>
-        <element name="selectedImagePreview" type="text" selector="//img[contains(@alt,'{{imageName}}')]" parameterized="true"/>
+        <element name="mediaGalleryImage" type="text" selector="//img[contains(@alt,'{{imageName}}')]" parameterized="true"/>
+        <element name="mediaGalleryDeleteButton" type="button" selector="#delete_files"/>
```

Comment:
@sivaschenko, I think these _can_ be moved to another Section since these two are not really related to Adobe Stock, but rather the Magento Media Gallery.

Reply:
@drpayyne that's a really good suggestion! Ideally, these selectors can be contributed to the CMS module of magento2 repository, but currently we have to keep them in our module. Moving to a separate section would be good.

Reply:
@sivaschenko @drpayyne should I create an issue for moving the selectors out of this project and into magento2? Possibly as part of Milestone 3 (after the next 2.3.4 release)?